lib/rollbax.ex

```diff
@@ -14,10 +14,13 @@ defmodule Rollbax do
 
   The following is a comprehensive list of configuration options supported by Rollbax:
 
-    * `:access_token` - (binary) the token needed to access the [Rollbar Items
+    * `:access_token` - the token needed to access the [Rollbar Items
       API (POST)](https://rollbar.com/docs/api/items_post/). As of now, Rollbar
       provides several access tokens for different "parts" of their API: for
       this configuration option, the "post_server_item" access token is needed.
+      This option requires a binary or `{:system, "VAR_NAME"}`. When given a
+      tuple like `{:system, "VAR_NAME"}`, the value will be referenced from
+      `System.get_env("VAR_NAME")` at runtime.
```

Comment:
I took this wording from phoenix's `Phoenix.Endpoint` documentation that also has the `{:system, var}` support.